codecept.conf.js

```diff
@@ -0,0 +1,24 @@
+const { setHeadlessWhen, setCommonPlugins } = require('@codeceptjs/configure');
+// turn on headless mode when running with HEADLESS=true environment variable
+// export HEADLESS=true && npx codeceptjs run
+setHeadlessWhen(process.env.HEADLESS);
+
+// enable all common plugins https://github.com/codeceptjs/configure#setcommonplugins
+setCommonPlugins();
+
+/** @type {CodeceptJS.MainConfig} */
+exports.config = {
+  tests: './*_test.js',
+  output: './output',
+  helpers: {
+    Playwright: {
+      browser: 'chromium',
+      url: 'http://localhost:8080',
+      show: true
+    }
+  },
+  include: {
+    I: './steps_file.js'
```

Comment:
```suggestion
    I: './src/tests/e2e/steps_file.ts'
```
해당 디렉토리로 경로 이동하고 ts로 변경 가능할까요?